assets/training/model_management/environments/foundation-model-inference/context/requirements.txt

```diff
@@ -1,17 +1,20 @@
-text-generation==0.6.0
+deepspeed~=0.10.0
```

Comment:
base image already comes with deepspeed, torch etc packages installed. Why are you installing again?

Reply:
If it comes with the required version then it would be a no-op we have tested with the versions satisfied with these requirements only.

Reply:
Also, we will evaluate later if we need this base image or something slimmer than this would do.

Reply:
i have been experimenting with nvidia's base image and we might switch to that in the next version (if not the one after that for sure)

Reply:
ok, can you create a workitem to track so it doesn't get missed?